pkg/solana/chainwriter/lookups.go

```diff
@@ -0,0 +1,271 @@
+package chainwriter
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+
+	ag_binary "github.com/gagliardetto/binary"
+	"github.com/gagliardetto/solana-go"
+	"github.com/gagliardetto/solana-go/rpc"
+
+	"github.com/smartcontractkit/chainlink-solana/pkg/solana/client"
+)
+
```

Comment:
```suggestion
// Lookup represents any mechanisms for resolving Solana accounts into `solana.AccountMeta`
```

Reply:
I went with something a little different 

`// Lookup is an interface that defines a method to resolve an address (or multiple addresses) from a given definition.
`